requirements-ci.txt

```diff
@@ -23,3 +23,4 @@ multi_quantization
 
 onnx
 onnxruntime
+onnx_graphsurgeon
```

Comment:
Could you fix the CI ?
https://github.com/k2-fsa/icefall/runs/7664079998?check_suite_focus=true
```
Collecting onnx_graphsurgeon
  Downloading onnx-graphsurgeon-0.0.1.dev5.tar.gz (7.9 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'error'
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [17 lines of output]
      Traceback (most recent call last):
        File "<string>", line 36, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-5dszyv1l/onnx-graphsurgeon_e8d8938e054143e78919b624c4c3d8c8/setup.py", line 150, in <module>
          raise RuntimeError(open("ERROR.txt", "r").read())
      RuntimeError:
      ###########################################################################################
      The package you are trying to install is only a placeholder project on PyPI.org repository.
      This package is hosted on NVIDIA Python Package Index.
      
      This package can be installed as:
      ```
      $ pip install nvidia-pyindex
      $ pip install onnx-graphsurgeon
      ```
      ###########################################################################################
      
      [end of output]
```